packages/auth/webpack.config.mjs

```diff
@@ -1,7 +1,7 @@
 import * as Repack from '@callstack/repack';
 import path from 'path';
 import TerserPlugin from 'terser-webpack-plugin';
-
+import getSharedDependencies from 'super-app-showcase-sdk/lib/shared-deps.js';
```

Comment:
we're importing possibly private `lib/` folder of this `super-app-showcase-sdk` module. Let's expose `getSharedDependencies ` as a public module API